docs/_react/intro.story.tsx

```diff
@@ -32,5 +32,24 @@ export const Intro = () => (
       format
       dark
     />
+    <p>
+      Then (only once) inline the <code>Icons</code> SVG sprite in your app:
```

Comment:
We seem to assume they are going to use icons.
Should we perhaps include language for "if you are going to use <Icon />" for this paragraph and for installing `@onfido/castor-icons`?

Reply:
Good point.

I will do this as separate PR, as it also includes docs for "core" part.